certified-connectors/Knowledgeone RecFind 6/apiDefinition.swagger.json

```diff
@@ -0,0 +1,376 @@
+{
+  "swagger": "2.0",
+  "info": {
+    "title": "Knowledgeone RecFind6",
+    "description": "Knowledgeone RecFind6 provides table query and file upload services from external system to RecFind 6 databases.",
+    "version": "1.0",
+    "contact": {
+      "name": "Knowledgeone Corporation",
+      "url": "https://knowledgeonecorp.com",
+      "email": "support@knowledgeonecorp.com"
+    }
+  },
+  "host": "example.com",
```

Comment:
@ecardakaris-knowledgeonecorp Since it is defined in a policy template, you can update host URL as "yourhost.yourdomain.com".